payments-core/src/main/java/com/stripe/android/analytics/SessionSavedStateHandler.kt

```diff
@@ -13,7 +14,28 @@ object SessionSavedStateHandler {
 
     private var sessionLocked: Boolean = false
 
-    fun startSession(savedStateHandle: SavedStateHandle) {
+    /**
+     * Attaches a [Session] to a [ViewModel]'s lifecycle and its provided [SavedStateHandle]. This should be called
+     * in the first line of the `init` call in the [ViewModel].
+     *
+     * @param viewModel used to attach a session to its lifecycle.
+     * @param savedStateHandle used to store & restore [Session] data.
+     *
+     * @return a function for manually restarting the session if needed.
+     */
+    fun attachTo(viewModel: ViewModel, savedStateHandle: SavedStateHandle): () -> Unit {
```

Comment:
I’m only seeing the return value used in a test. Is there no use for it in production?